extensions/EXT/SPV_EXT_physical_storage_buffer.asciidoc

```diff
@@ -131,6 +152,29 @@ to:
     "If the *VariablePointers* or *VariablePointersStorageBuffer* <<Capability,capability>>
     is declared, the following are allowed for <<LogicalPointerType,logical pointer types>>:".
 
+
+Change:
+
+    *OpVariable* can allocate an object whose type is a pointer type, if the
+    <<Storage_Class, Storage Class>> of the *OpVariable* is one of the
+    following: ...
+
+to:
+
+    *OpVariable* can allocate an object whose type is a
+    <<LogicalPointerType,logical pointer type>>, if the
+    <<Storage_Class, Storage Class>> of the *OpVariable* is one of the
```

Comment:
BTW, mark-up wise, this is 'Storage Class', as it's meant to be that operand of the instruction.  Adding operand helps:

...if the 'Storage Class' operand of the *OpVariable*...

That's what makes it unambiguous about levels of indirection involved.

Reply:
(I think the extension spec. is okay; the link doesn't work anyway.)